File: piwik-sdk/src/test/java/testhelpers/BaseTest.java
package testhelpers;

import org.junit.After;
import org.junit.Before;

import timber.log.Timber;


public class BaseTest {

    @Before
    public void setup() throws Exception {
        Timber.plant(new JUnitTree());
    }

    @After
    public void tearDown() throws Exception {
        Timber.uprootAll();
    }
}

File: tracker/src/test/java/testhelpers/BaseTest.kt
package testhelpers

import org.junit.After
import org.junit.Before
import timber.log.Timber
import timber.log.Timber.Forest.plant

open class BaseTest {
    @Before
    @Throws(Exception::class)
    open fun setup() {
        plant(JUnitTree())
    }

    @After
    @Throws(Exception::class)
    open fun tearDown() {
        Timber.uprootAll()
    }
}
